src/web/services/userAccount.ts

```diff
@@ -26,11 +23,11 @@ export type InviteTeamMemberForm = {
 export type UpdateTeamMemberForm = Omit<InviteTeamMemberForm, 'email'>;
 
 export type UserPayload = z.infer<typeof UserCreationPartial>;
-export type UserResponse = UserWithCurrentParticipantRoleNames;
+export type UserResponse = UserWithParticipantRoles;
```

Comment:
Do we think this `UserResponse` still makes sense? I find it a bit confusing - it's used in a lot of places that aren't directly calling an API and getting a response. 

At the very least, perhaps we could replace a bunch of usages with `UserWithParticipantRoles`. But perhaps we should remove this type altogether - I think it's just adding a layer of abstraction that we don't need.